fastapi_users/manager.py

```diff
@@ -110,6 +110,7 @@ async def get_by_oauth_account(self, oauth: str, account_id: str) -> models.UP:
     async def create(
         self,
         user_create: schemas.UC,
+        background_tasks: BackgroundTasks,
         safe: bool = False,
         request: Optional[Request] = None,
```

Comment:
Adding the `background_tasks` as the last and optional function parameter.

```suggestion
        safe: bool = False,
        request: Optional[Request] = None,
        background_tasks: Optional[BackgroundTasks] = None,
```